ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/match-lists.j2

```diff
@@ -5,13 +5,13 @@
 {%     if match_list_input.string is arista.avd.defined %}
 ### Match-list Input String Summary
 
-{%         for match_list in match_list_input.string | arista.avd.natural_sort %}
-#### {{ match_list }}
+{%         for match_list in match_list_input['string'] | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
```

Comment:
please use `match_list_input.string` instead of `match_list_input['string']`. Same in eos template.